sonar-scanner-engine/src/main/java/org/sonar/scanner/phases/PhasesTimeProfiler.java

```diff
@@ -44,10 +50,25 @@ public void onSensorsPhase(SensorsPhaseEvent event) {
   @Override
   public void onSensorExecution(SensorExecutionEvent event) {
     if (event.isStart()) {
-      profiler.startInfo("Sensor " + ScannerUtils.describe(event.getSensor()));
+      ClassLoader cl = getSensorClassLoader(event.getSensor());
+      String pluginKey = pluginRepo.getPluginKey(cl);
+      String suffix = "";
+      if (pluginKey != null) {
+        suffix = " [" + pluginKey + "]";
+      }
+      profiler.startInfo("Sensor " + ScannerUtils.describe(event.getSensor()) + suffix);
     } else {
       profiler.stopInfo();
     }
   }
 
+  private ClassLoader getSensorClassLoader(Sensor sensor) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Make "getSensorClassLoader" a "static" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2325)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this use of "Sensor"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)